eclipse-collections-forkjoin/pom.xml

```diff
@@ -54,8 +54,21 @@
         <dependency>
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>
+            <scope>test</scope>
+        </dependency>
+	<!-- 
+        <dependency>
+            <groupId>org.slf4j</groupId>
```

Comment:
Why is this dependency being added?

Reply:
Let me revert this as this was unintentional (I wasn't able skip the tests)